### PR TITLE
Add instalation instructions to MacOS

### PR DESCRIPTION
**Pull request by Coquinho**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -134,3 +138,82 @@ vcpkg-script folder in the windows-dependencies repository.
...
+        $ export DYLD_LIBRARY_PATH="${DYLD_LIBRARY_PATH}:${VCPKG_FOLDER}/vcpkg/installed/x64-osx/debug/lib"
+        $ export DYLD_LIBRARY_PATH="${DYLD_LIBRARY_PATH}:${DESKTOP_PREFIX}/lib/debug"
+```
+You will need these variables every time you open up a new terminal. So you may put the exports on a file, here `setenv`, to just source it:
+```bash
+        $ source setenv
+```
+### Building the repository
+
+To build the repository clone `expertisesolution/dali-core`, change to `walac/macos` branch and enter the 'build/tizen' folder:
```
Comment (walac) [inline README.md]: Remove references from expertise fork.

**Review by walac — CHANGES_REQUESTED** [review]

**Review by walac — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -134,3 +138,92 @@ vcpkg-script folder in the windows-dependencies repository.
...
+```
+### Build the repository
+
+To build the repository enter the 'build/tizen' folder:
+```bash
+         $ cd dali-core/build/tizen
+```
+Then run the following command to set up the build:
+```bash
+         $ cmake -DCMAKE_INSTALL_PREFIX=$DESKTOP_PREFIX -DCMAKE_EXPORT_COMPILE_COMMANDS=ON -Wno-dev -DCMAKE_TOOLCHAIN_FILE=$VCPKG_FOLDER/vcpkg/scripts/buildsystems/vcpkg.cmake -DINSTALL_CMAKE_MODULES=ON
```
Comment (walac) [inline README.md]: `-DCMAKE_EXPORT_COMPILE_COMMANDS=ON` and `-Wno-dev` are not necessary.

**Review by walac — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (Coquinho) [reply README.md]: I've removed it. If you think they should be mentioned I can add some exeplanations on them.

File: README.md [hunk 3]
```diff
@@ -134,3 +138,92 @@ vcpkg-script folder in the windows-dependencies repository.
...
+ - `zlib`
+
+And from `homebrew` we need:
+ - `cairo`
+ - `cmake`
+
+### Set a DALi Environment
+Now you need to create a `dali-env` folder and then set some environment variables.
+First of all:
+- `VCPKG_FOLDER` should contain the absolute path to where is your `vcpkg` instalation
```
Comment (walac) [inline README.md]: nit: installation (with double l)

**Review by walac — CHANGES_REQUESTED** [review]

**Comment by walac** [comment]
12 files changed?

**Comment by Coquinho** [comment]
> 12 files changed?

`walac/macos` was force-pushed, so now this branch needs a rebase

**Comment by Coquinho** [comment]
As rebase was needed so I took the chance to squash commits too.

**Review by walac — APPROVED** [review]

**Pull request closed without merging**